typescript/package.json

```diff
@@ -1,21 +1,35 @@
 {
-  "name": "hathora-cloud-sdk",
+  "name": "@hathora/hathora-cloud-sdk",
```

Comment:
```suggestion
  "name": "@hathora/cloud-sdk",
```

Reply:
so sid and i have been talking about this for a few days--we think that this makes sense because it's "hathora's sdk for the hathora-cloud product" not hathora's sdk for "cloud"